dev-lua/luarocks/luarocks-2.4.2.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="A deployment and management system for Lua modules"
+DESCRIPTION="LuaRocks allows you to install modules as self-contained \
+packages called rocks, which also contain version dependency information."
+HOMEPAGE="http://luarocks.org/"
+COPYRIGHT="2007-2016 Kepler Project"
+LICENSE="MIT"
+REVISION="2"
+SOURCE_URI="http://luarocks.org/releases/luarocks-$portVersion.tar.gz"
+CHECKSUM_SHA256="0e1ec34583e1b265e0fbafb64c8bd348705ad403fe85967fd05d3a659f74d2e5"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	luarocks = $portVersion
+	cmd:luarocks = $portVersion
+	cmd:luarocks_5.3 = $portVersion
+	cmd:luarocks_admin = $portVersion
+	cmd:luarocks_admin_5.3 = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:liblua
+	"
+BUILD_REQUIRES="
+	haiku_devel
+	devel:liblua
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:ld
+	cmd:make
+	cmd:md5sum
+	cmd:sed
+	cmd:wget
+	"
+
+BUILD()
+{
+	#not an autotools package
+	./configure --prefix=$prefix  \
+		--with-lua-include="$includeDir/lua53" \
```

Comment:
This should do the trick, as discussed on IRC :)
LUA_VERSION=$(lua -v | sed 's/Lua //' | head -c3 | tr -d '.')
./configure --prefix=$prefix --with-lua-include="$(finddir B_SYSTEM_DEVELOP_DIRECTORY)/headers/lua$LUA_VERSION"